components/inputs/test/input-number.test.js

```diff
@@ -411,6 +411,13 @@ describe('d2l-input-number', () => {
 				expect(elem.invalid).to.equal(!!test.expectedError);
 			});
 		});
+
+		it('should be invalid when required and value is less than min', async() => {
+			const elem = await fixture(html`<d2l-input-number label="label" min="5" required value="0"></d2l-input-number>`);
+			await elem.updateComplete;
```

Comment:
Is this line needed? Our wrapper around `fixture` should be waiting for that already.